### PR TITLE
[Perfomance][export] Opt the export of CSV tranformer

### PR DESCRIPTION
**Pull request by HappenLee**
## Proposed changes

Before：
```
mysql [tpch]>select * from customer INTO OUTFILE "file:///mnt/disk1/baizhenbo/run/data/customer";
+------------+-----------+------------+-------------------------------------------------------------------------------------------+
| FileNumber | TotalRows | FileSize   | URL                                                                                       |
+------------+-----------+------------+-------------------------------------------------------------------------------------------+
|          3 |  15000000 | 2448490271 | file:///127.0.0.1/mnt/disk1/baizhenbo/run/data/customerf420bb41b05d4e87-869f2f93f475e773_ |
+------------+-----------+------------+-------------------------------------------------------------------------------------------+
1 row in set (20.53 sec)
```

After：
```
mysql [tpch]>select * from customer INTO OUTFILE "file:///mnt/disk1/baizhenbo/run/data/customer";
+------------+-----------+------------+-------------------------------------------------------------------------------------------+
| FileNumber | TotalRows | FileSize   | URL                                                                                       |
+------------+-----------+------------+-------------------------------------------------------------------------------------------+
|          3 |  15000000 | 2448490271 | file:///127.0.0.1/mnt/disk1/baizhenbo/run/data/customerc582a3aad60745f8-bfeeda617b66a7f2_ |
+------------+-----------+------------+-------------------------------------------------------------------------------------------+
1 row in set (14.74 sec)
```

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by HappenLee** [comment]
run buildall

**Comment by HappenLee** [comment]
run buildall

**Comment by HappenLee** [comment]
run buildall

**Review by Gabriel39 — APPROVED** [review]
LGTM

File: be/src/vec/runtime/vcsv_transformer.cpp [hunk 1]
```diff
@@ -130,45 +135,45 @@ Status VCSVTransformer::write(const Block& block) {
                     // For example: For a double value 27361919854.929001,
                     // the direct output of using std::stringstream is 2.73619e+10,
                     // and after conversion to a string, it outputs 27361919854.929001
-                    char buffer[MAX_DOUBLE_STR_LENGTH + 2];
+                    char buffer[MAX_DOUBLE_STR_LENGTH + 2] = "\0";
                     double double_value =
                             *reinterpret_cast<const double*>(col.column->get_data_at(i).data);
                     buffer[0] = '\0';
```
Comment (liutang123) [inline be/src/vec/runtime/vcsv_transformer.cpp:141]: This line can be removed.

**Review by liutang123 — APPROVED** [review]
LGTM

**Comment by HappenLee** [comment]
run buildall

**Review by liutang123 — APPROVED** [review]
LGTM

**Pull request merged**